cli/ldap.go

```diff
@@ -179,3 +182,122 @@ func DeleteLdap(c *cli.Context) error {
 	log.Infof("[DeleteLdap] ldap config deleted: %s", ldapName)
 	return nil
 }
+
+func CreateLdapUser(c *cli.Context) error {
+	checkRequiredFlagsAndArguments(c)
+	defer utils.TimeTrack(time.Now(), "create ldap user")
+
+	directoryType := c.String(FlLdapDirectoryType.Name)
+	validateDirectoryTypeToManageUsers(directoryType)
+
+	bindUser := c.String(FlLdapBindDN.Name)
+	bindPassword := c.String(FlLdapBindPassword.Name)
+	ldapServer := c.String(FlLdapServer.Name)
+	userName := c.String(FlLdapUserToCreate.Name)
+	password := c.String(FlLdapUserToCreatePassword.Name)
+	baseDn := c.String(FlLdapUserToCreateBase.Name)
+	groups := c.String(FlLdapUserToCreateGroups.Name)
+
+	ldap, err := ldaputils.Dial("tcp", ldapServer)
+	if err != nil {
+		utils.LogErrorAndExit(err)
+	}
+	defer ldap.Close()
+
+	log.Infof("[CreateLdapUser] connecting to LDAP %s with bind %s", ldapServer, bindUser)
+	err = ldap.Bind(bindUser, bindPassword)
+	if err != nil {
```

Comment:
I suggest to use "inline" check: `if err = ldap.Bind(bindUser, bindPassword); err != nil {`